sql/03_create-view.sql

```diff
@@ -0,0 +1,48 @@
+CREATE OR REPLACE VIEW api.events_view AS
+SELECT
+    id,
+
+    -- Schema detection (identify unique markers)
+    CASE
+        WHEN event->>'source_system' = 'LTT' THEN 'ltt'
+        WHEN event->>'legacy_id' IS NOT NULL THEN 'legacy'
+        WHEN event->>'loggerName' IS NOT NULL THEN 'ltt'
+        WHEN event->>'asctime' IS NOT NULL THEN 'legacy'
+        ELSE 'unknown'
```

Comment:
I think null might be a bit more customary

```suggestion
        ELSE 'null'
```